### ISSUE TITLE
tkdesigner 1.0.4 requires requests==2.25.1, but you have requests 2.27.1 which is incompatible.

### ISSUE THREAD

**Issue by Mennaruuk** [ParthJadhav/Tkinter-Designer#186]
Dear Parth,

I hope you are feeling well.

I would love to know if you can change the requirements for Tkinter Designer so that versions of `requests` higher than 2.25.1 are compatible. Unfortunately, with the latest `requests` 2.27.1 version, I cannot install Tkinter Designer, and I get the following error:

`tkdesigner 1.0.4 requires requests==2.25.1, but you have requests 2.27.1 which is incompatible.`

Thank you very much.

**Comment by AndhikaWB** [comment]
I agree with this, it's kinda annoying. I sometimes need to use other Python module which requires newer version of requests. Adding compatibility with request v2.27.1 (current version) would be lovely.
